StreamChatUITestsAppUITests/Robots/UserRobot+Asserts.swift

```diff
@@ -143,6 +143,20 @@ extension UserRobot {
         }
         return self
     }
+    
+    func assertComposerDoesNotGrowMoreThanFiveLines(file: StaticString = #filePath, line: UInt = #line) {
+        let composer = MessageListPage.Composer.inputField
+        var composerHeight = composer.height
+        for i in 1...4 {
+            let obtainKeyboardFocus = (i == 1) ? true : false
+            typeText("\(i)\n", obtainKeyboardFocus: obtainKeyboardFocus)
+            let updatedComposerHeight = composer.height
+            XCTAssertGreaterThan(updatedComposerHeight, composerHeight, file: file, line: line)
+            composerHeight = updatedComposerHeight
+        }
+        typeText("5\n6", obtainKeyboardFocus: false)
+        XCTAssertEqual(composerHeight, composer.height, file: file, line: line)
+    }
```

Comment:
Can we pass the `lines` in the argument? Something like `assertComposerDoesNotGrow(moreThan numberOfLines:)`, right now if our UI Logic changes, we will need to change the function name, and the implementation as well